pkgs/development/tools/trunk/default.nix

```diff
@@ -20,9 +20,8 @@ rustPlatform.buildRustPackage rec {
   };
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = if stdenv.hostPlatform.isDarwin
-    then [ CoreServices SystemConfiguration ]
-    else [ openssl ];
+  buildInputs = [ openssl ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [ CoreServices SystemConfiguration ];
```

Comment:
```suggestion
  buildInputs = [ openssl ];
```

Try this.

Reply:
tldr: `apple_sdk`s are added by default now

Reply:
Done, and confirmed that it works - thanks!